src/communication/azext_communication/tests/latest/recordings/test_list_phonenumbers.yaml

```diff
@@ -0,0 +1,46 @@
":"inbound+outbound"},"assignmentType":"application","purchaseDate":"2021-08-04T09:49:45.3765116+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18334241267","phoneNumber":"+18334241267","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"inbound+outbound","sms":"inbound+outbound"},"assignmentType":"application","purchaseDate":"2021-04-06T23:29:30.1082746+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18334511835","phoneNumber":"+18334511835","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"inbound+outbound","sms":"inbound+outbound"},"assignmentType":"application","purchaseDate":"2021-04-08T19:23:21.6246128+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18334521095","phoneNumber":"+18334521095","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"inbound+outbound","sms":"inbound+outbound"},"assignmentType":"application","purchaseDate":"2021-04-08T19:23:49.1286852+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18334861434","phoneNumber":"+18334861434","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"none","sms":"outbound"},"assignmentType":"application","purchaseDate":"2021-04-12T08:20:46.6769231+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18334861443","phoneNumber":"+18334861443","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"none","sms":"outbound"},"assignmentType":"application","purchaseDate":"2021-04-12T13:14:11.6087977+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18334861445","phoneNumber":"+18334861445","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"outbound","sms":"inbound+outbound"},"assignmentType":"application","purchaseDate":"2021-04-12T13:18:16.5746747+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18334970824","phoneNumber":"+18334970824","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"none","sms":"outbound"},"assignmentType":"application","purchaseDate":"2021-04-12T15:41:20.7171509+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18335022077","phoneNumber":"+18335022077","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"inbound","sms":"inbound+outbound"},"assignmentType":"application","purchaseDate":"2021-04-13T12:13:04.2873741+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}},{"id":"18335022090","phoneNumber":"+18335022090","countryCode":"US","phoneNumberType":"tollFree","capabilities":{"calling":"inbound","sms":"inbound+outbound"},"assignmentType":"application","purchaseDate":"2021-04-13T12:15:50.8017707+00:00","cost":{"amount":2.0,"currencyCode":"USD","billingFrequency":"monthly"}}]}'
```

Comment:
Need to sanitize phone numbers as well.
We should sanitize phone numbers to be phone numbers that follow the format 1-<areacode>-555-<remaining 4 digits>.
so for example, this one could be +15015550256
This is to prevent the sanitized number being a real phone number